### PR TITLE
Updaded NEST nuget package to 7.8.2

### PR DESCRIPTION
**Pull request by edg-jf**
Updaded NEST nuget package to 7.8.2
Updated Unit and Integration Tests

### CODE REVIEW

File: Source/EventFlow.Elasticsearch.Tests/IntegrationTests/ElasticsearchReadModelStoreTests.cs [hunk 1]
```diff
@@ -92,37 +92,47 @@ private void PrepareIndexes(IRootResolver resolver)
...
+                _elasticClient.Indices.Create(indexName, c => c
                     .Settings(s => s
                         .NumberOfShards(1)
                         .NumberOfReplicas(0))
-                    .Aliases(a => a.Alias(esType.Name))
-                    .Mappings(m => m
-                        .Map(TypeName.Create(readModelType), d => d
-                            .AutoMap())));
+                    .Aliases(a => a.Alias(esType.RelationName)));
+                    // I do not believe this is still needed? If yes, I am unsure how to pass the Type into the mapping method
```
Comment (rasmus) [inline Source/EventFlow.Elasticsearch.Tests/IntegrationTests/ElasticsearchReadModelStoreTests.cs:132]: If you automatic mapping, it is. I seem to remember that for 7.x you can pass the type. If you can't find it, I'll give it a go

**Review by rasmus — CHANGES_REQUESTED** [review]
Just a minor comment and if you could update the release notes, specifically in regards to the wire version. I can't remember which major version of Elasticsearch that the 7.8 client supports.

Update the image used for integration tests as well

File: Source/EventFlow.Elasticsearch.Tests/IntegrationTests/ElasticsearchReadModelStoreTests.cs (hunk 1 above)
Comment (frenchyjef) [reply Source/EventFlow.Elasticsearch.Tests/IntegrationTests/ElasticsearchReadModelStoreTests.cs:132]: @rasmus Happy to update the release notes but I am unsure what exactly you are asking me to change. 

Updated NEST nuget package to 7.8.2 (targets Elastic Search >= 7.8.2) 

Is this what you would like me to change?

Regarding Integration Test image, I do not know what that means or how to change it... could you give me instructions?

Regarding Auto Mapping, I did spend a considerable amount of time trying to pass a generic type but could not find a way to make it work. Your help would be appreciated.

Comment (frenchyjef) [reply Source/EventFlow.Elasticsearch.Tests/IntegrationTests/ElasticsearchReadModelStoreTests.cs:132]: @rasmus Quick follow up on my above comments... do you have any guidance so we can finalize this pull request?

Comment (rasmus) [reply Source/EventFlow.Elasticsearch.Tests/IntegrationTests/ElasticsearchReadModelStoreTests.cs:132]: Sorry, I have been buried in work and haven't had any time to look at EventFlow. I'll have a look at this tonight

Comment (rasmus) [reply Source/EventFlow.Elasticsearch.Tests/IntegrationTests/ElasticsearchReadModelStoreTests.cs:132]: Could you add a line with something like this to the release notes

- Updated Elasticsearch NEST from v6.1.0 to v7.8.2

As for the un-commented lines in the test around automap, just delete them.

Comment (frenchyjef) [reply Source/EventFlow.Elasticsearch.Tests/IntegrationTests/ElasticsearchReadModelStoreTests.cs]: Pull request updated with your feedback. Let me know if any other changes required.

**Comment by rasmus** [comment]
Perfect, I'll merge as soon as https://github.com/eventflow/EventFlow/issues/795 has been resolved.

**Review by rasmus — APPROVED** [review]

**Pull request merged**